exotations/exotica_core_task_maps/src/eff_box.cpp

```diff
@@ -76,6 +81,43 @@ Eigen::VectorXd EffBox::GetUpperLimit() const
     return eff_upper_;
 }
 
+void EffBox::PublishObjectsAsMarkerArray()
+{
+    const ros::Time t = ros::Time::now();
+    visualization_msgs::MarkerArray ma;
+    ma.markers.reserve(n_effs_);
+    for (int i = 0; i < n_effs_; ++i)
+    {
+        const int eff_id = 3 * i;
+        visualization_msgs::Marker m;
+        m.header.stamp = t;
+        std::string frame_name;
+        if (frames_[i].frame_B_link_name == "")
+        {
+            frame_name = "world_frame";
```

Comment:
`world_frame` or `exotica/world_frame`, or more correct - get the actual name (if someone renames their init_ros("something") it will affect it...)

Reply:
Use `Scene::GetRootFrameName()`. I think that `exotica/` is a predefined prefix at the moment.

Reply:
@VladimirIvan `exotica/` prefix is required, see [here](https://github.com/ipab-slmc/exotica/blob/28fe61c0afa91ab2fe9b025105131b1698d86bba/exotations/exotica_core_task_maps/src/quasi_static.cpp#L346).